src/reconcile/rec_write.c

```diff
@@ -1687,7 +1688,8 @@ __rec_child_modify(WT_SESSION_IMPL *session,
 			 */
 			if (F_ISSET(r, WT_EVICTING)) {
 				WT_ASSERT(session, !F_ISSET(r, WT_EVICTING));
-				return (EBUSY);
+				ret = EBUSY;
+				goto done;
```

Comment:
We generally use an error label when there is an error condition being flagged. In this case it'd be better to replace this with:

```
- ret = EBUSY;
- goto done;
+ WT_ERR(EBUSY);
```

Then you could add the `error:` label to the statistics increment line, and keep the semantic identical.